addons/point_of_sale/static/src/app/components/popups/product_configurator_popup/product_configurator_popup.js

```diff
@@ -128,8 +128,33 @@ export class ProductConfiguratorPopup extends Component {
             productTemplate: this.props.productTemplate,
             product: null,
             payload: this.env.attribute_components,
+            info: null,
         });
 
+        useEffect(
+            () => {
+                const fetchProductInfo = async () => {
+                    if (!this.props.productTemplate) {
+                        return;
+                    }
+
+                    try {
+                        const productInfo = await this.pos.getProductInfo(
+                            this.props.productTemplate,
+                            1
+                        );
+                        // Update state with fetched product info
+                        this.state.info = productInfo;
+                    } catch (error) {
+                        console.error("Error fetching product info:", error);
+                    }
+                };
+
+                fetchProductInfo(); // Call the async function inside useEffect
+            },
+            () => [this.props.productTemplate]
+        ); // Dependencies array: trigger when productTemplate changes
```

Comment:
Fetching price was interesting if it change by selecting different variant (set as always created). Since this feature didn't works anymore, i'm wondering if we should keep that fetch.

See

```js
        useRefListener(this.inputArea, "touchend", this.computeProductProduct.bind(this));
        useRefListener(this.inputArea, "click", this.computeProductProduct.bind(this));
 ```

All data to compute these information are available localy in the case of product_configurator_popup